Public/Src/Cache/ContentStore/Distributed/NuCache/TransitioningContentLocationStore.cs

```diff
@@ -275,6 +278,8 @@ public Task<BoolResult> UnregisterAsync(Context context, IReadOnlyList<ContentHa
                     priorityQueue.Push(lastAccessTimes[i]);
                 }
 
+                context.Debug($"GetLruPages page.First:(Age={page.First().Age}, EffectiveAge={page.First().EffectiveAge}), queue.Top:(Age={priorityQueue.Top.Age}, EffectiveAge={priorityQueue.Top.EffectiveAge})");

```

Comment:
>First [](start = 49, length = 5)

Can we print the info the last item of the page as well? Maybe include hash of first and last item of page as well.